code/modules/research/techweb/_techweb_node.dm

```diff
@@ -109,10 +113,12 @@
 			for(var/booster in boostlist)
 				if(actual_costs[booster])
 					actual_costs[booster] -= boostlist[booster]
-		for(var/cost_type in actual_costs)
-			for(var/experiment_type in discount_experiments)
-				if(host.completed_experiments[experiment_type]) //do we have this discount_experiment unlocked?
-					actual_costs[cost_type] -= discount_experiments[experiment_type]
+		if(!applied_discount)
+			for(var/cost_type in actual_costs)
+				for(var/experiment_type in discount_experiments)
+					if(host.completed_experiments[experiment_type]) //do we have this discount_experiment unlocked?
+						actual_costs[cost_type] -= discount_experiments[experiment_type]
+						applied_discount = TRUE
```

Comment:
Doesn't this mean that once one discount is applied, no more can be? even if more experiments are completed?

Reply:
Now that I think about it, it will, actually. My original thoughts were about adding the var to the experiment in question, but I didn't quite figure out how to do that. I'll look into it a bit more.